2200075/2_2001 . 파리 퇴치.py

```diff
@@ -0,0 +1,23 @@
+T = int(input())
+for t in range(1,T+1) :
+
+    N, M = map(int,input().split())
+
+    matrix = [list(map(int,input().split())) for n in range(N)]
+
+    # for y1 in range(N -M) :
+    #     for x1 in range(N -M) :
+    result = []
+    matrix2 = [[0*y for y in range(M)] for x in range(M)]
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    matrix2 = [[0 for y in range(M)] for x in range(M)]
```
불필요한 연산은 지워주세요 😄